docs/pages/api-docs/menu.md

```diff
@@ -56,8 +56,8 @@ Any other props supplied will be provided to the root element ([Popover](/api/po
 |:-----|:-------------|:------------|
 | <span class="prop-name">paper</span> | <span class="prop-name">.MuiMenu-paper</span> | Styles applied to the `Paper` component.
 | <span class="prop-name">list</span> | <span class="prop-name">.MuiMenu-list</span> | Styles applied to the `List` component via `MenuList`.
-| <span class="prop-name">disablePointerEvents</span> | <span class="prop-name">.MuiMenu-disablePointerEvents</span> | Styles applied to subMenus to prevent the Modal from capturing mouse events
```

Comment:
is running `yarn docs:api` enough?

Reply:
yeah, I was having a disconnect between the JSDoc comments in the d.ts vs the ones in the js. I think I got it. I just wanted to make sure those internal APIs weren't documented in the api docs

Reply:
We've got a fix for the test coming shortly, I believe.